src/client/pages/OfferNew/utils.ts

```diff
@@ -33,7 +35,7 @@ export const getOfferData = (quoteBundle: QuoteBundle): OfferData => {
       ssn: firstQuote.ssn,
       phoneNumber: firstQuote.phoneNumber,
       birthDate: firstQuote.birthDate,
-      householdSize: getHouseholdSizeFromBundledQuotes(quoteBundle.quotes),
+      ...{ householdSize },
```

Comment:
just curious why this object destructing is needed?

Reply:
Seems to work totally fine as it was, probably some refactoring that got stuck =) Will revert!